src/core/p5.Renderer2D.js

```diff
@@ -621,24 +621,19 @@ class Renderer2D extends p5.Renderer{
         return this;
       }
     }
-    const kappa = 0.5522847498,
-      // control point offset horizontal
-      ox = w / 2 * kappa,
-      // control point offset vertical
-      oy = h / 2 * kappa,
-      // x-end
-      xe = x + w,
-      // y-end
-      ye = y + h,
-      // x-middle
-      xm = x + w / 2,
-      ym = y + h / 2; // y-middle
+    const xm = x + w / 2, // x-middle
+      ym = y + h / 2, // y-middle
+      radiusX = w / 2,
+      radiusY = h / 2;
     if (!this._clipping) ctx.beginPath();
-    ctx.moveTo(x, ym);
-    ctx.bezierCurveTo(x, ym - oy, xm - ox, y, xm, y);
-    ctx.bezierCurveTo(xm + ox, y, xe, ym - oy, xe, ym);
-    ctx.bezierCurveTo(xe, ym + oy, xm + ox, ye, xm, ye);
-    ctx.bezierCurveTo(xm - ox, ye, x, ym + oy, x, ym);
+    // ctx.moveTo(x, y);
```

Comment:
Nice work! Can we take out the commented-out old implementation before merging?

Reply:
sure thing
I will also finish other functions and update the PR for review

Reply:
resolved it here https://github.com/processing/p5.js/pull/6499/commits/b73af847cc0c949630710fa162e1f95316cd3d63